src/System.Text.Json/src/System/Text/Json/Serialization/Converters/DefaultConverters.cs

```diff
@@ -2,12 +2,27 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Concurrent;
 using System.Reflection;
 
 namespace System.Text.Json.Serialization.Converters
 {
     internal static class DefaultConverters
     {
+        private static readonly ConcurrentDictionary<Type, object> _valueConverters = new ConcurrentDictionary<Type, object>();
+
+        static DefaultConverters()
+        {
+            _valueConverters.TryAdd(typeof(DateTimeOffset), new JsonValueConverterDateTimeOffset());
+            _valueConverters.TryAdd(typeof(Guid), new JsonValueConverterGuid());
+            _valueConverters.TryAdd(typeof(JsonElement), new JsonValueConverterJsonElement());
```

Comment:
Can this dictionary be initialized with a field initializer instead of a cctor?

```C#
private static readonly ConcurrentDictionary<Type, object> _valueConverters = new ConcurrentDictionary<Type, object>
{
    { typeof(DateTimeOffset), new JsonValueConverterDateTimeOffset() },
    { typeof(JsonElement), new JsonValueConverterJsonElement() },
    { typeof(Guid), new JsonValueConverterGuid() },
};
```

Or something to that effect?

Reply:
Unfortunately no as there isn't a `Add` method on `ConcurrentDictionary`.

Reply:
It can be a Dictionary instead of a ConcurrentDictionary.  Dictionary is safe to be read from concurrently by any number of threads, as long as it's not being mutated concurrently, and in this case, it's not mutated after it's initialized.